pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -60,9 +60,13 @@
       </template>
       <template #col:containerName="{ row }">
         <td>
-          <span v-tooltip="getTooltipConfig(row.containerName)">
+          <a
+            v-tooltip="getTooltipConfig(row.containerName)"
+            class="container-name-link"
+            @click.stop.prevent="navigateToInfo(row)"
```

Comment:
Any reason this doesn't use `viewInfo` instead? The two functions seem to be duplicates.